saleor/payment/utils.py

```diff
@@ -365,3 +367,14 @@ def is_currency_supported(currency: str, gateway_id: str, manager: "PluginsManag
     """Return true if the given gateway supports given currency."""
     available_gateways = manager.list_payment_gateways(currency=currency)
     return any([gateway.id == gateway_id for gateway in available_gateways])
+
+
+def get_last_payment(payments: List[Payment]):
```

Comment:
It looks like it returns `last_payment` for the order but we never check if the payment is valid.  Not sure if this a proper way